docs/contributor/network/multus.md

```diff
@@ -0,0 +1,204 @@
+# Multus in MicroShift
+
+MicroShift 4.16 includes an optional Multus CNI.
+It can be used to attach additional interfaces to Pods.
+
+## How to install
+
+Manifests to deploy Multus CNI are part of `microshift-multus` RPM.
+Depending on the timeline it can be installed from OpenShift mirror or built locally.
+Specific instructions depend on source of the RPM.
```

Comment:
Multus depends on the exact version of MicroShift which is already installed. I think we need to rephrase these to make it clearer.